README.md

```diff
@@ -3,8 +3,8 @@
 ![Rust](https://github.com/hello-rust/lychee/workflows/Rust/badge.svg)
 [![docs.rs](https://docs.rs/lychee/badge.svg)](https://docs.rs/lychee)
 
-⚡ A fast, async, resource-friendly link checker written in Rust.  
-Finds broken hyperlinks and mail addresses inside Markdown, HTML, reStructuredText, or any other text file or website!  
+⚡ A fast, async, resource-friendly link checker written in Rust.
```

Comment:
Can we add back the two spaces at the end? They are significant because they result in a newline, which is the formatting I prefer in that case.

Reply:
They're done by my formatter...